languages/python/oso/tests/test_oso.py

```diff
@@ -129,5 +130,26 @@ def test_allow_model(test_oso):
     assert test_oso.is_allowed(user, "list", Company)
 
 
+def test_get_allowed_actions(test_oso):
+    rule = """allow(_actor: test_oso::Actor{name: "Sally"}, action, _resource: test_oso::Widget{id: "1"}) if
+        action in ["CREATE", "READ"];"""
+
+    test_oso.load_str(rule)
+    user = Actor(name="Sally")
+    resource = Widget(id="1")
+    assert set(test_oso.get_allowed_actions(user, resource)) == set(
+        ["get", "CREATE", "READ"]
```

Comment:
where did "get" come from?

Reply:
From a different rule that already existed in the test policy

Reply:
Slightly unfortunate. If there's no easy way to make the test independent, then it's not worth trying to address.

Reply:
It's not that hard to change, but would rather take it as a follow-up.